Vizzuality/marxan-cloud
# create new CostSurface and CostSurfacePuData entities in dbs [MRXN23-93]

##### Creates CostSurface and CostSurfacePuData entities in dbs

Creates a new entity in each DB

NOTE: Commit tree clean-up pending.


### Overview

Adds a new table in each DB via 2 new migrations, along with it's entity representation.


### Testing instructions

Check that once you run the app, those new tables are created, and the required constraints are in place

### Feature relevant tickets

[MRXN23-93](https://vizzuality.atlassian.net/browse/MRXN23-93)
---

## Checklist before submitting

      deploying to staging/production, please add brief testing instructions
      to the deploy checklist (`docs/deployment-checklist.md`)

[MRXN23-93]: https://vizzuality.atlassian.net/browse/MRXN23-93?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
